src/meta/src/stream/stream_graph/fragment.rs

```diff
@@ -896,11 +896,17 @@ impl CompleteStreamFragmentGraph {
         } = building_fragment;
 
         let distribution_type = distribution.to_distribution_type() as i32;
+        let materialized_fragment_id =
```

Comment:
`building_fragment.extract_internal_tables()` doesn't return the state table of `Materialize` node?